### ISSUE TITLE
Clicking Thumbnail on Map Does Not Expand Image

### ISSUE THREAD

**Issue by gitrormeister** [TomoyukiAota/photo-location-map#451]
Not critical, but here are my specs:
Win10Pro 21H1 Build 19043.1526

Sample image attached.
![IMG_8229](https://user-images.githubusercontent.com/8978499/197582108-ab12d7b0-1232-4737-a363-cb3313f5e62e.JPG)

**Comment by TomoyukiAota** [comment]
@gitrormeister 
On my PC (Win10Pro 21H2 Build 19044.2130), the app associated with .jpg extension opens just like the attached recording.
https://user-images.githubusercontent.com/25931120/199271244-6eb0c8f8-6620-437e-a82e-237397a8afb3.mp4
(In the recording, the flash of the yellow circle is given by the recording tool to indicate the mouse click.)

Clicking the thumbnail opens the file with the associated program, which is the same behavior as double clicking the file in Explorer.
Could you check if the file can be opened by double click in Explorer on your PC?

**Comment by TomoyukiAota** [comment]
To reply to https://github.com/TomoyukiAota/photo-location-map/discussions/452#discussioncomment-4028899, the default app associated to .jpg extension needs to be able to open the file. This app will simply launch the default app and cannot support opening the file if the default app cannot open the file.

Also, please use this issue thread for this issue. https://github.com/TomoyukiAota/photo-location-map/discussions/452 is a different topic.

**Comment by TomoyukiAota** [comment]
Please find the app that can open your .jpg file, and set it as a default app to open .jpg files.

**Comment by gitrormeister** [comment]
Yes, that is what I did. Originally my default jpg app was FastStone Image
Viewer. Since that wasn’t working I reset the default to M$ Photos, but it
had now affect.

Thanks
